src/components/TextField/README.md

```diff
@@ -737,6 +737,36 @@ Use to show that a textfield is not available for interaction. Most often used i
 <TextField label="Store name" disabled />
 ```
 
+### Text field with character count
+
+<!-- example-for: web -->
+
+Use to show the number of characters remaining in a text field with a max length. If max length is not provided it will display the current number of characters in the text field.
```

Comment:
Same comment as I've already left, but I find this a bit unclear.

Reply:
```suggestion
Use to show the number of characters remaining in a text field with a max length. If max length is not provided it will display the current number of characters in the text field.
```